deploy/kubernetes/base/controller.env

```diff
@@ -0,0 +1 @@
+CONTROLLER_PROVISIONER_TIMEOUT=5m
```

Comment:
What's the value of creating a separate file for this vs. putting it in the values.yaml:
https://github.com/kubernetes-sigs/aws-fsx-csi-driver/blob/master/charts/aws-fsx-csi-driver/values.yaml#L42-L67

Reply:
This file is for manifests only for kustomize. `values.yaml` is dedicated to helm chart. I understand this driver supports both kustomize and helm.

In kustomize, injecting parameter in kustomize needs a bit hack. This env file is needed for kustomize users to change timeout value. I also updated `install.md` as below:

https://github.com/everpeace/aws-fsx-csi-driver/blob/suppor-dra/docs/install.md#deploy-driver
```
# To set CSI controller's provisioner timeout,
# Please follow the instruction
$ cd $(mktemp -d)
$ kustomize init
$ kustomize edit add resource "github.com/kubernetes-sigs/aws-fsx-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.1"
$ kustomize edit add configmap fsx-csi-controller --from-literal=CONTROLLER_PROVISIONER_TIMEOUT=30m --behavior=merge
$ kubectl apply -k .
```

Reply:
I think we should avoid hacks when possible and this seems like an avoidable instance. If users want to configure their kustomize templates, they can download them, configure them, and deploy them freely. We should follow precedent in terms of implementation, which is to put it in the values.yaml.